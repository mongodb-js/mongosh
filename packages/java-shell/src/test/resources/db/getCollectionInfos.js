// before
db.testCollection1.remove({});
db.testCollection1.insertOne({a: 1});
// command
db.getCollectionInfos({name: {$regex: "testCollection.*"}});
// clear
db.testCollection1.drop();
