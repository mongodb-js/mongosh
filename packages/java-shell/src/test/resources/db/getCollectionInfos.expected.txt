[ { "name": "testCollection1", "type": "collection", "options": { }, "info": { "readOnly": false, "uuid": <UUID> }, "idIndex": { "v": 2, "key": { "_id": 1 }, "name": "_id_", "ns": "test.testCollection1" } } ]